pkg/statdb/sdbclient/client.go

```diff
@@ -120,7 +130,8 @@ func (sdb *StatDB) FindValidNodes(ctx context.Context, nodeIDs [][]byte, minAudi
 }
 
 // Update is used for updating a node's stats in the stats db
-func (sdb *StatDB) Update(ctx context.Context, nodeID []byte, auditSuccess, isUp bool, latencyList []int64,
+func (sdb *StatDB) Update(ctx context.Context, nodeID []byte, 
+	auditSuccess, isUp bool, latencyList []int64,
```

Comment:
--nit
same as above

Reply:
I added a separate `UpdateUptime` function so what I think I will do is make another one called `UpdateAudit`, meaning each of those can be used if you want to update one attribute and not the other. Then we can assume anyone calling `Update` wants to update all attributes, which removes a bunch of args

Reply:
Actually I'll just add a jira ticket for this